osquery/config/plugins/tls.cpp

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
I'm still working on this whole approach. But for now I renamed all enrollment -> enroll.